src/lib/types/Types.ts

```diff
@@ -14,6 +14,11 @@ type TimelineData = {
 	background_image?: Image;
 	images: Array<Image>;
 	content: Array<RichtextElement | RichtextTextElement>;
+	devprops?: {
```

Comment:
`TimelineData` is the type of mapped data. So we need to map the `devprop` from the cms response in `src/routes/+page.server.ts`. And since we need to map it to `TimelineData`, we may also just map it to a `vfx: string` or `vfx: 'confetti'`.

Reply:
Thanks, that certainly makes one of my checks a lot cleaner.